pkg/processor/runtime/golang/runtime.go

```diff
@@ -37,6 +38,8 @@ type golang struct {
 
 	// TODO: support multiple cancels when implementing async
 	cancelEventHandlingChan chan context.CancelFunc
+	// TODO: remove when implement full safe statistics for async processing
+	statisticsMutex sync.RWMutex
```

Comment:
iirc statistics are atomic, why do we need mutex here?

Reply:
@liranbg no, they are uints - https://github.com/nuclio/nuclio/blob/7c8736dae1951a312a2343fb2d1f7233dcb3e12f/pkg/processor/runtime/types.go#L32-L35